src/main/plugins/org.dita.base/plugin.xml

```diff
@@ -172,6 +172,78 @@ See the accompanying LICENSE file for applicable license.
       <val>true</val>
       <val default="true">false</val>
     </param>
+    <param name="build-step.clean-temp" desc="Run process clean-temp" type="enum">
```

Comment:
This could be `build-step.cleanup`?

Reply:
Maybe even `build-step.cleanup-temp` ... not sure if "cleanup" would be more readable, but that might be because I've been setting `clean-temp` for 15 years. So maybe cleanup + temp (clear that it's cleaning some temp dir)?

Reply:
I think we can leave it to be `clean-temp` in this PR just because it's more readable for us for the mean time. All the descriptions etc. need to be sorted and we can rename it then.